montecarlo.py

```diff
@@ -178,6 +193,38 @@ def simulate(self):
 ##        self.list_cache = list_cache
 ##        return self.list_cache
 
+    def simulateTL(self,timestep): #Only works for first timestep
+        """Simulates the Monte Carlo simulation on the Cayley Tree for one
+           time step and stores that data."""
+        time_steps = range(len(self.state_d)) 
+        if self.list_cache == None:
+            list_cache = list()
+            list_cache.append(self.state_d)
+        else:
+            list_cache = self.list_cache
+        cache = dict()
+        no_nodes = (self.network.links*(self.network.links-1)**(self.network.generations-1))
+        if timestep == 0:
+            dens = 0

```

Comment:
What is the dens variable? Please throw a comment after or rename if a better name can be found